README.md

```diff
@@ -114,6 +114,16 @@ ln -sf "${THEME_DIR}/gtk-4.0/gtk.css" "${HOME}/.config/gtk-4.0/gtk.css" &&
 ln -sf "${THEME_DIR}/gtk-4.0/gtk-dark.css" "${HOME}/.config/gtk-4.0/gtk-dark.css"
 ```
 
+## Flatpak
+> [!WARNING]
+> Flatpak is deprecated and we don't maintain it anymore,
+> We don't recommend themeing Flatpak with the theme.
```

Comment:
Its not "deprecated", we are just not the team that maintains it. We don't mind giving flatpak access to the files. We just don't advice using the flatpak package.

Reply:
oh